mlflow/deployments/cli.py

```diff
@@ -451,3 +453,46 @@ def get_endpoint(target, endpoint):
     for key, val in desc.items():
         click.echo(f"{key}: {val}")
     click.echo("\n")
+
+
+try:
+    from mlflow.gateway.config import _validate_config
```

Comment:
Can we import this lazily in `validate_config_path`?

Reply:
Yeah, I think that's better. Thanks - done!